.github/workflows/cover.yml

```
@@ -0,0 +1,33 @@
+name: Coverage
+on: [pull_request]
+jobs:
+  build:
+    name: Build
```

Maybe it is better to call the job Coverage instead of Build
```suggestion
  coverage:
    name: Coverage
```